serverless.yml

```diff
@@ -19,6 +19,12 @@ provider:
     CLIENT_ID: ${env:CLIENT_ID}
     CLIENT_SECRET: ${env:CLIENT_SECRET}
     RANGE: ${env:RANGE}
+  iamRoleStatements:
```

Comment:
Move this to a lambda-specific role, see example from booster. Use serverless references instead of hard-coded ARNs